### PR TITLE
Harkinians: add imgui.ini adjustments during launch

### PR DESCRIPTION
**Pull request by JeodC**
This change modifies `imgui.ini` during launch to ensure all imgui windows stay within the display bounds.

### CODE REVIEW

File: ports/soh2/Ship of Harkinian 2.sh [hunk 1]
```diff
@@ -15,6 +15,8 @@ fi
 source $controlfolder/control.txt
 get_controls
 
+# Source Device Info
+source $controlfolder/device_info.txt
 [ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
```
Comment (cdeletre) [inline ports/soh2/Ship of Harkinian 2.sh:21]: Applying the new launch script template 

```suggestion
[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
get_controls
```

File: ports/soh2/Ship of Harkinian 2.sh [hunk 2]
```diff
@@ -90,11 +123,11 @@ fi
 
 # Run the game
 echo "Loading, please wait... (might take a while!)" > $CUR_TTY
-$GPTOKEYB "2s2h.elf" -c "soh2.gptk" &
-pm_platform_helper 2s2h.elf
+$GPTOKEYB "2s2h.elf" -c "soh2.gptk" &
```
Comment (cdeletre) [inline ports/soh2/Ship of Harkinian 2.sh:126]: Maybe there is a good reason to remove it, just tell me :)
```suggestion
pm_platform_helper "2s2h.elf"
$GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
```

File: ports/soh2/Ship of Harkinian 2.sh [hunk 3]
```diff
@@ -90,11 +123,11 @@ fi
...
+$GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
 ./2s2h.elf
 
 # Cleanup
 rm -rf "$GAMEDIR/logs/"
-
-pm_finish
+$ESUDO systemctl restart oga_events & 
+printf "\033c" >> /dev/tty1
+printf "\033c" > /dev/tty0
```
Comment (cdeletre) [inline ports/soh2/Ship of Harkinian 2.sh:133]: Same remark, just tell me if I'm wrong :)
```suggestion
pm_finish
```

**Review by cdeletre — CHANGES_REQUESTED** [review]
Not sure why soh2 is like reversing to the old sh launch template. Any good reason ? :)

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**